azerothcore/azerothcore-wotlk
# refactor(Core/SpellInfo): Positive/Negative spells from SpellFamilyName into DB

## CHANGES PROPOSED:
* Implement SPELL_ATTR0_CU_POSITIVE_EFF0

* Implement SPELL_ATTR0_CU_POSITIVE_EFF1

* Implement SPELL_ATTR0_CU_POSITIVE_EFF2

* Move SpellFamilyName spells into DB to define if they are POSITIVE or NEGATIVE spells

* Rename column from entry to spell_id to make more sense

## TESTS PERFORMED:
- Builds
- Tested in-game (works)



## HOW TO TEST THE CHANGES:

1. .aura 61716
2. See that this is a Positive spell (Appears as a Buff)
3. .aura 34709
4. See that this is a Negative spell (Appears as a Debuff)

This means that the new Attribute works.


## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
